restapi/dataset/item.avsc

```diff
@@ -4,15 +4,10 @@
   "name": "Item",
   "doc": "Dataset item. It contains a value and its related time.",
   "fields": [
-    {"name": "value", "type": [
-      "org.radarcns.avro.restapi.sensor.HeartRate" ,
-      "org.radarcns.avro.restapi.sensor.Acceleration" ,
-      "org.radarcns.avro.restapi.sensor.Battery" ,
-      "org.radarcns.avro.restapi.sensor.BloodVolumePulse" ,
-      "org.radarcns.avro.restapi.sensor.ElectroDermalActivity" ,
-      "org.radarcns.avro.restapi.sensor.InterBeatInterval" ,
-      "org.radarcns.avro.restapi.sensor.Temperature"
-      ] } ,
-    {"name": "effectiveTimeFrame", "type": "org.radarcns.avro.restapi.header.EffectiveTimeFrame"}
+    {"name": "sample", "type": [
+      "org.radarcns.avro.restapi.data.DoubleSample" ,
```

Comment:
Aha ok. Btw: I'm still not sure how to interpret this quality in an aggregated statistic. Suppose I have a descriptive statistic `MIN` over time period T and it has quality 0.6, is that also the minimum quality over that period T? Or would we send the same mean quality value over T for every descriptive statistic `MIN`, `MAX`, `SUM`, etc.?